src/components/tables/table-pagination/table-pagination.tsx

```diff
@@ -9,6 +9,9 @@ export const TablePagination = (props: MuiTablePaginationProps) => {
   return (
     <MuiTablePagination
       {...props}
+      // it seems like this should work as:  component="div"
+      //   maybe a bug in material-ui's typescript types, moving on for now
+      component={({ children }: { children?: React.ReactNode }) => <div>{children}</div>}
```

Comment:
You might be able to do this. `Box` is material-ui's basic div equivalent.
```suggestion
      component={Box}
```